content/english/python-basics/_index.md

```diff
@@ -1,14 +1,31 @@
 ---
 title: "Python: Basics"
 description: "Learn basic concepts of using the Python programming language"
-date: 2019-07-30T18:45:38-07:00
+date: 2022-09-19T14:45:38-07:00
 prereq: "none"
 difficulty: "Beginner"
 icon: "fab fa-python"
 draft: false
 ---
 
 # Python basics
+
+## Overview
+
+Welcome to the course <b>Python: Basics</b>, where you will learn a widely-used, high-level programming language that gives computer instructions to create exciting programs!
+
+Python is a high-level programming language created by Guido van Rossum between 1985-1990. 
+
+{{% notice info %}}
+A high-level programming language is called high-level because it is quite a few steps away from the original code run on a computer's Central Processing Unit (CPU). High-level code is meant to be used by humans and is much easier to understand than low-level code. The high-level code is later translated into a low-level language, which can only be perceived and run by a specific CPU. For the code to work, every single line of code written in a high-level language must be translated into machine language before being implemented by the computer.
```

Comment:
Maybe we could mark this as optional reading? Don't think younger students will be interested in this